src/isar/state_machine/states/monitor.py

```diff
@@ -110,17 +145,12 @@ def _queue_inspections_for_upload(self, current_step: InspectionStep) -> None:
             inspections: Sequence[
                 Inspection
             ] = self.state_machine.robot.get_inspections(step=current_step)
-        except Exception as e:
-            if self.state_machine.stepwise_mission:
-                self.logger.error(
-                    f"Error getting inspections for step "
-                    f"{str(current_step.id)[:8]}: {e}"
-                )
-            else:
-                self.logger.error(
-                    f"Error getting inspections for mission "
-                    f"{str(self.state_machine.current_mission.id)[:8]}: {e}"
-                )
+
+        except (RobotRetrieveInspectionException, RobotException) as e:
```

Comment:
Might be enough to except RobotException as RobotRetrieveInspectionException inherits from RobotException

Reply:
Same argument as above.

Reply:
Fine by me